treasury-donation/README.md

```diff
@@ -0,0 +1,54 @@
+
+This means that to allow companies to donate to the treasury, we need to define an on-chain address for the treasury. This can be done following the same format as the [upgradeable treasury smart contract](https://github.com/dcSpark/plutus-smart-contract-upgrade-dApp) from dcSpark.
+
+### Updating the treasury
+
+Cardano does not have an on-chain governance method for deciding protocol parameters. This missing functionality also makes it hard to decide on how to upgrade the treasury contract. As a temporary workaround similar to the current behavior, we could add MIR certificates to the Plutus context so that the contract can look at the context to decide if the contract can be upgraded.
+
+### Convincing projects to pitch in
+
+Once private ventures receive Catalyst funding, there is nothing to force them to actually donate back to the treasury after they've achieved market success. This should instead be encouraged at the social level and Catalyst proposals for these categories should be encouraged to define in their proposal whether they have any plans to donate funds back to the treasury.
+
+Additionally, the community should encourage projects to pitch in with assets that are as uncorrelated with ADA as possible (different baskets)
+
+### Compatibility with future treasury plans
+
+The upgradeability of the contract should make it easy to keep this system compatible with existing plans for the Cardano treasury. For clarity, I will detail some of the treasury plans and how this proposal is compatible with them:
+
+1. **Allowing Catalyst rounds in non-ADA tokens** (ex: COTI rewards for COTI proposals which has happened in the past). This proposal is compatible with this idea because projects could send their tokens to the treasury address. Additionally, it makes itt easier for this to happen because we could allow projects to put in 10K ADA (arbitrary number) as a fee to mint a project ID to the treasury address which can be redeemable once to add the project to Catalyst using the funds for that token present in the address. The fee of this mint would be used to both fund the main ADA treasury balance and to cover the cost of distributing rewards for the Catalyst challenge of the project.
+2. **DCF**. DCF is a more complicated version of Catalyst meant for higher amounts with higher auditability on spending. DCF plans to have its own Plutus scripts, so there may be an overlap in the approach, but these DCF scripts could receive their initial funding from the treasury address so there should not be a compatibility issue.
```

Comment:
The best source I could find on **DCF** was the video from last year's Cardano Summit.  If there's anything written on the subject it would help to link it here, so people can see how the ideas are compatible without having to go through the video. 😎